templates/zerver/register.html

```diff
@@ -148,21 +148,23 @@ <h1>{{ _('Create your account') }}</h1>
                 </div>
 
                 {% if require_ldap_password %}
-                <div class="input-box">
-                    <input id="ldap-password" class="required" type="password" name="password" required />
+                <div class="input-box password-div">
+                    <input id="ldap-password" class="required" type="password" name="password" autocomplete="off" required />
```

Comment:
I think this one is correct to be autocomplete=off, since we're asking for the current LDAP password, not a new password.  (In contrast with the next one in this file).